async/index.native.js

```diff
@@ -1,8 +1,8 @@
-import { getRandomBytesAsync } from 'expo-random'
+import * as Crypto from 'expo-crypto'
```

Comment:
Can we avoid importing everything?

```suggestion
import { getRandomBytesAsync } from 'expo-crypto'
```

Reply:
Just updated the code :)